internal/install/discovery/os_validator.go

```diff
@@ -0,0 +1,33 @@
+package discovery
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/newrelic/newrelic-cli/internal/install/types"
+)
+
+type OsValidator struct{}
+
+var (
+	NoOperatingSystemDetected         = "failed to identify a valid operating system"
```

Comment:
Do we need to export these variables? i.e. if not, they can start with a lowercase letter.

Reply:
Good catch, they can remain internal. I did several try/error with my test and forgot to loop back to hide those.
I've implemented the suggested change.